app/views/layouts/application.html.haml

```diff
@@ -14,7 +14,7 @@
     %meta{:name => "author", :content => "Diaspora, Inc."}/
 
     %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
-    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
+    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
```

Comment:
@SuperTux88 : Alright then. I won't delete more images than I already did. It's only about 5 images so I don't think you will have much trouble when rebasing against develop when this is merged.

About the favicon, I think that a rounded version, a bit smaller than the actual apple icon could fit all the needs. The logo is simple enough to be resized to a smaller scale without felling weird.